test/extended/builds/forcepull.go

```diff
@@ -79,18 +87,14 @@ we built.  The credentials also are recycled between those points.
 Dumping of the ImageStreams and Secrets JSON output at the various points proved this out.
 */
 
-var _ = g.Describe("[LocalNode][builds] forcePull should affect pulling builder images", func() {
+var _ = g.Describe("[builds] forcePull should affect pulling builder images", func() {
 	defer g.GinkgoRecover()
 	var oc = exutil.NewCLI("forcepull", exutil.KubeConfigPath())
 
 	g.BeforeEach(func() {
 
-		g.By("refresh corruptor, prep forcepull builder")
-		exutil.PullImage(corruptor, dockerClient.AuthConfiguration{})
-
-		exutil.DumpImage(corruptor)
-
 		// create the image streams and build configs for a test case specific builders
+		g.By("create image stream and builder build config")
 		setupPath := exutil.FixturePath("testdata", "forcepull-setup.json")
```

Comment:
Missed this nuance from the earlier exchange, but yeah, goes along with the fallout of no longer corrupting images.  I'll take a look.